doc/user_guide/dynamo0p3.rst

```diff
@@ -786,50 +792,63 @@ Supported Function Spaces
 **continuity** of a function space (horizontal, vertical or full).
-Alternatively, if there are no shared dofs a function space is described
+Alternatively, if there are no shared DoFs a function space is described
 as **discontinuous** (fully or in a particular direction).
 
 The mixed FEM formulation is built on a foundation set of four function
 spaces described below.
 
-* ``w0`` is the space of scalar functions with full continuity. The
-  shared dofs lie on cell vertices in the lowest order FEM and on
+* ``W0`` is the space of scalar functions with full continuity. The
+  shared DoFs lie on cell vertices in the lowest order FEM and on
   all three entities in higher order FEM.
 
-* ``w1`` is the space of vector functions with full continuity in the
-  tangential direction only. In the lowest order FEM the shared dofs
+* ``W1`` is the space of vector functions with full continuity in the
+  tangential direction only. In the lowest order FEM the shared DoFs
   lie on cell edges for each component, whereas in higher order they
   also lie on cell faces.
 
-* ``w2`` is the space of vector functions with full continuity in the
-  normal direction only. The shared dofs lie on cell faces for each
+* ``W2`` is the space of vector functions with full continuity in the
+  normal direction only. The shared DoFs lie on cell faces for each
   component.
 
-* ``w3`` is the space of scalar functions with full discontinuity.
-  All dofs lie within the cell volume and are not shared across the
+* ``W3`` is the space of scalar functions with full discontinuity.
+  All DoFs lie within the cell volume and are not shared across the
   cell boundaries.
 
 Other spaces required for representation of scalar or component-wise
 vector variables are:
 
-* ``wtheta`` is the space of scalar functions based on the vertical
-  part of ``w2``, discontinuous in the horizontal and continuous
+* ``Wtheta`` is the space of scalar functions based on the vertical
+  part of ``W2``, discontinuous in the horizontal and continuous
+  in the vertical;
+
+* ``W2H`` is the space of vector functions based on the horizontal
+  part of ``W2``, continuous in the horizontal and discontinuous
   in the vertical;
 
-* ``w2v`` is the space of vector functions based on the vertical
-  part of ``w2``, discontinuous in the horizontal and continuous
+* ``W2V`` is the space of vector functions based on the vertical
+  part of ``W2``, discontinuous in the horizontal and continuous
   in the vertical;
 
-* ``w2h`` is the space of vector functions based on the horizontal
-  part of ``w2``, continuous in the horizontal and discontinuous
-  in the vertical.
+* ``W2broken`` is the space of vector functions, locally identical
+  to the velocity space ``W2``. However, DoFs are topologically
+  discontinuous in all directions despite their placement on cell
+  faces;
+
+* ``W2trace`` is the space of scalar functions, resulting from
+  taking the ``trace`` of a ``W2`` element and defined only on the
```

Comment:
This was Thomas Gibson's description. It is not completely mathematically correct but it is a very close short description of what is actually happening. `W2` space is defined as a tensor product of its basis functions and `W2trace` results from taking the trace of the `W2` tensor product so only the "diagonal" scalar components remain instead of the usual `W2` 3D velocity components.
I modified the description and removed "element" so it is hopefully a bit clearer now, however I am not really sure how much sense it makes to go into more details.